hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -337,7 +337,26 @@ stop_docker_env(){
   copy_daemon_logs
   save_container_logs
   if [ "${KEEP_RUNNING:-false}" = false ]; then
-     docker-compose --ansi never down
+    down_repeats=3
+    result=0
+    for i in $(seq 1 $down_repeats)
+    do
+      docker-compose --ansi never down
+      result=$?
+      if [[ $result -eq 0 ]]
+      then
+        echo "Docker-compose down successful"
+        break
+      else
+        echo "Error while trying to shut down docker containers"
+        sleep 5
+      fi
+    done
+
+    if [[ $result -ne 0 ]]
+    then
+      echo "Couldn't remove all docker containers after $down_repeats repeats."
+    fi
```

Comment:
I think we should fail the test run if `docker-compose down` is unsuccessful even after several retries.  I guess this cannot happen due to the intermittent lag in stopping containers, and is most likely due to some error in the test script.

```suggestion
    echo "Failed to remove all docker containers in $down_repeats attempts."
    return 1
```